apps/smart-contracts/core/test/unit/DepositHook.test.ts

```diff
@@ -201,6 +201,29 @@ describe('=> DepositHook', () => {
       expect(tokenSender.send).not.called
     })
 
+    it("doesn't call tokenSender.send() if fee > 0 but reward = 0", async () => {
```

Comment:
```suggestion
    it("doesn't call tokenSender.send() if multiplier = 0", async () => {
```
(fee != 0 is implied from above test)

same for other hooks